pkg/kubelet/kubelet.go

```diff
@@ -2892,6 +2893,12 @@ func (kl *Kubelet) handlePodResourcesResize(pod *v1.Pod, podStatus *kubecontaine
 		if err := kl.statusManager.SetPodAllocation(pod); err != nil {
 			return nil, err
 		}
+		for i, container := range pod.Spec.Containers {
```

Comment:
are only main containers resizeable? initContainers / ephemeralContainers aren't?

Reply:
Only main containers & sidecars, but sidecar support hasn't merged yet (coming in https://github.com/kubernetes/kubernetes/pull/128367)